test/OpenTelemetry.Exporter.LightStep.Tests/OpenTelemetry.Exporter.LightStep.Tests.csproj

```diff
@@ -8,7 +8,7 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
-        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.6.0-preview7.19362.9" />
+        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.6.0" />
```

Comment:
this reference should be removed at all - it is coming through OpenTelemetry.Abstractions